src/redux/store.js

```diff
@@ -1,11 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
-import booksReducer from './books/booksSlice';
-import categoriesReducer from './categories/categoriesSlice';
+import booksSlice from './books/booksSlice';
+import categoriesSlice from './categories/categoriesSlice';
 
 const store = configureStore({
   reducer: {
-    books: booksReducer,
-    categories: categoriesReducer,
+    books: booksSlice.reducer,
+    categories: categoriesSlice.reducer,
   },
 });
```

Comment:
- [ ] Your use of Redux is a step in the right direction; however, it seems there might be a configuration issue in your current setup, leading to errors as displayed in the screenshot .
![image](https://github.com/BenMKT/Bookstore/assets/55840999/f3362bfd-066e-4acd-9732-3115c251684c)
To resolve this, I recommend following the official [Redux Toolkit tutorial](https://redux-toolkit.js.org/tutorials/overview). This tutorial will provide you with detailed guidance on configuring your Redux store correctly, ensuring a smooth and error-free integration with your application. By aligning your Redux setup with best practices, you'll not only fix the current errors but also ensure a more robust and maintainable codebase. Keep up the great work! 🚀📦🔧

Reply:
- Refactor all files to use Redux in React components to solve bugs.